lib/active_merchant/billing/gateways/pay_trace.rb

```diff
@@ -187,10 +187,15 @@ def acquire_access_token
           'Content-Type'      => 'application/x-www-form-urlencoded'
         }
         response = ssl_post(url, data, oauth_headers)
-        json_response = JSON.parse(response)
+        json_response = parse(response)
```

Comment:
Why remove the `JSON.`?

Reply:
Just cause parse does the same thing lol